### PR TITLE
Event Listeners (part 1)

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: ???

### Summary of Changes

* I paired with @quodlibetor to create a minimal event listeners UI
* I added support for adding/removing associated breakpoints as well
* The styling and UX is all very prototypy and will definitely change

### Test Plan

- [ ] check to see if the list is complete
- [ ] does the checkboxes / close button work as expected
- [ ] make sure it's flagged off :)

### CODE REVIEW

**Comment by quodlibetor** [comment]
Is "the list" in the first checkbox "the list of event listeners"?

**Comment by jasonLaster** [comment]
yeah, the list is the list of event listeners

File: src/components/RightSidebar.js [hunk 1]
```diff
@@ -72,6 +73,14 @@ const RightSidebar = React.createClass({
       { header: L10N.getStr("scopes.header"),
         component: Scopes }
     ];
+
+    if (isEnabled("eventListeners")) {
+      items.push({
+        header: "EventListeners",
```
Comment (wldcordeiro) [inline src/components/RightSidebar.js]: Here we'd want `Event Listeners` instead no (probably an L10N string would be better.)

Comment (jasonLaster) [inline src/components/RightSidebar.js]: Yea, good call

**Pull request merged**